core/src/test/scala/com/databricks/labs/remorph/parsers/common/FunctionBuilderSpec.scala

```diff
@@ -191,243 +191,248 @@ class FunctionBuilderSpec extends AnyFlatSpec with Matchers with TableDrivenProp
-      ("SESSION_CONTEXT", Some(FunctionDefinition.standard(1, 2))),
-      ("SESSION_USER", Some(FunctionDefinition.standard(0))),
-      ("SESSIONPROPERTY", Some(FunctionDefinition.standard(1))),
-      ("SIGN", Some(FunctionDefinition.standard(1))),
-      ("SIN", Some(FunctionDefinition.standard(1))),
-      ("SMALLDATETIMEFROMPARTS", Some(FunctionDefinition.standard(5))),
-      ("SOUNDEX", Some(FunctionDefinition.standard(1))),
-      ("SPACE", Some(FunctionDefinition.standard(1))),
-      ("SQL_VARIANT_PROPERTY", Some(FunctionDefinition.standard(2))),
-      ("SQRT", Some(FunctionDefinition.standard(1))),
-      ("SQUARE", Some(FunctionDefinition.standard(1))),
-      ("STATS_DATE", Some(FunctionDefinition.standard(2))),
-      ("STDEV", Some(FunctionDefinition.standard(1))),
-      ("STDEVP", Some(FunctionDefinition.standard(1))),
-      ("STR", Some(FunctionDefinition.standard(1, 3))),
-      ("STRING_AGG", Some(FunctionDefinition.standard(2, 3))),
-      ("STRING_ESCAPE", Some(FunctionDefinition.standard(2))),
-      ("STUFF", Some(FunctionDefinition.standard(4))),
-      ("SUBSTRING", Some(FunctionDefinition.standard(2, 3))),
-      ("SUM", Some(FunctionDefinition.standard(1))),
-      ("SUSER_ID", Some(FunctionDefinition.standard(0, 1))),
-      ("SUSER_NAME", Some(FunctionDefinition.standard(0, 1))),
-      ("SUSER_SID", Some(FunctionDefinition.standard(0, 2))),
-      ("SUSER_SNAME", Some(FunctionDefinition.standard(0, 1))),
-      ("SWITCHOFFSET", Some(FunctionDefinition.standard(2))),
-      ("SYSDATETIME", Some(FunctionDefinition.standard(0))),
-      ("SYSDATETIMEOFFSET", Some(FunctionDefinition.standard(0))),
-      ("SYSTEM_USER", Some(FunctionDefinition.standard(0))),
-      ("SYSUTCDATETIME", Some(FunctionDefinition.standard(0))),
-      ("TAN", Some(FunctionDefinition.standard(1))),
-      ("TIMEFROMPARTS", Some(FunctionDefinition.standard(5))),
-      ("TODATETIMEOFFSET", Some(FunctionDefinition.standard(2))),
-      ("TOSTRING", Some(FunctionDefinition.standard(0))),
-      ("TRANSLATE", Some(FunctionDefinition.standard(3))),
-      ("TRIM", Some(FunctionDefinition.standard(1, 2))),
-      ("TYPE_ID", Some(FunctionDefinition.standard(1))),
-      ("TYPE_NAME", Some(FunctionDefinition.standard(1))),
-      ("TYPEPROPERTY", Some(FunctionDefinition.standard(2))),
-      ("UNICODE", Some(FunctionDefinition.standard(1))),
-      ("UPPER", Some(FunctionDefinition.standard(1))),
-      ("USER", Some(FunctionDefinition.standard(0))),
-      ("USER_ID", Some(FunctionDefinition.standard(0, 1))),
-      ("USER_NAME", Some(FunctionDefinition.standard(0, 1))),
-      ("VALUE", Some(FunctionDefinition.xml(2))),
-      ("VAR", Some(FunctionDefinition.standard(1))),
-      ("VARP", Some(FunctionDefinition.standard(1))),
-      ("XACT_STATE", Some(FunctionDefinition.standard(0))),
-      ("YEAR", Some(FunctionDefinition.standard(1))))
+      ("ABS", FunctionDefinition.standard(1)),
```

Comment:
Not sure what we are testing here ? Not behavior. I suggest we get rid of this test altogether...